src/web/builds.rs

```diff
@@ -27,10 +28,7 @@ impl Serialize for Build {
         state.serialize_field("rustc_version", &self.rustc_version)?;
         state.serialize_field("cratesfyi_version", &self.cratesfyi_version)?;
         state.serialize_field("build_status", &self.build_status)?;
-        state.serialize_field(
-            "build_time",
-            &time::at(self.build_time).rfc3339().to_string(),
-        )?;
+        state.serialize_field("build_time", &self.build_time.format("%+").to_string())?;
```

Comment:
What does `%+` mean? Can you add that as a comment?

Reply:
`%+` just means a RFC 3339 timestamp, it's in the [chrono docs](https://docs.rs/chrono/0.4.11/chrono/format/strftime/index.html)

Reply:
Let's add a comment anyway in case the docs or behavior ever change.